src/types/numeric.md

```diff
@@ -40,3 +40,8 @@ differences between pointers into an object or array and can address every byte
 within an object along with one byte past the end.
 
 `usize` and `isize` are at least 16-bits wide.
+
+> **Note**: Many pieces of Rust code may assume that pointers, `usize`, and
+> `isize` are at least 32-bit and at most 64-bit. As a consequence, 16-bit
```

Comment:
```suggestion
> `isize` are either 32-bit or 64-bit. As a consequence, 16-bit
```

Based on the discussion with @chorman0773. I think it's safe to say that these kinds of assumptions in the ecosystem would create similar difficulties for a target with pointer size *between* 32 and 64 bits.